demo/molecule/tabs/playground

```diff
@@ -1,189 +1 @@
-        />
-        <br/><br/>
-        <h2>HIGHLIGHTED VARIANT</h2>
-        <h3>Default no icons</h3>
-        <MoleculeTabs
-            items={itemsNoIcons}
-            activeTab={0}
-            handleChange={handleChange}
-            variant={moleculeTabsVariants.HIGHLIGHTED}
-        />
-        <br/><br/>
-        <h3>Default with icons</h3>
-        <MoleculeTabs
-            items={items}
-            activeTab={0}
-            handleChange={handleChange}
-            variant={moleculeTabsVariants.HIGHLIGHTED}
-        />
-        <br/><br/>
-        <h3>Full Width</h3>
-        <MoleculeTabs
-            items={items}
-            activeTab={0}
-            handleChange={handleChange}
-            variant={moleculeTabsVariants.HIGHLIGHTED}
-            type={moleculeTabsTypes.FULLWIDTH}
-        />
-        <br/><br/><br/><br />
-        <h1>Vertical Type</h1>
-        <div style={{
-          width: '100%',
-          float: 'left'
-        }}>
-          <table style={{
-              minWidth: '300px',
-              maxWidth: '50%',
-              float: 'left',
-              marginBottom: '20px'
-            }}>
-            <tr>
-              <td colSpan="2">
-                <h2>CLASSIC VARIANT</h2>
-              </td>
-            </tr>
-            <tr>
-              <td style={{
-                  width: '50%',
-                  verticalAlign: 'top'
-                }}>
-                <MoleculeTabs
-                    items={items}
-                    activeTab={0}
-                    handleChange={handleChange}
-                    type={moleculeTabsTypes.VERTICAL}
-                />
-              </td>
-            </tr>
-          </table>
-          <table style={{
-              minWidth: '300px',
-              maxWidth: '50%',
-              float: 'left',
-              marginBottom: '20px'
-            }}>
-            <tr>
-              <td colSpan="2">
-                <h2>HIGHLIGHTED VARIANT</h2>
-              </td>
-            </tr>
-            <tr>
-              <td style={{
-                  width: '50%',
-                  verticalAlign: 'top'
-                }}>
-                <MoleculeTabs
-                    items={items}
-                    activeTab={0}
-                    handleChange={handleChange}
-                    type={moleculeTabsTypes.VERTICAL}
-                    variant={moleculeTabsVariants.HIGHLIGHTED}
-                />
-              </td>
-            </tr>
-          </table>
-        </div>
-        <div style={{
-          width: '100%',
-          float: 'left'
-        }}>
-          <h1>Other features</h1>
-          <h3>Handle click in disabled tabs</h3>
-          <MoleculeTabs
-              items={itemsNoIcons}
-              activeTab={0}
-              handleChange={handleChange}
-              handleClickInDisabledTabs
-          />
-          <br/><br/>
-        </div>
-    </div>
-)
+return <p>This shouldn't be displayed</p>
```

Comment:
¿?

Reply:
Because, the folder `demo` should take control. I've noticed in some cases, when there's an error in `demo` this `playground` content is displayed
So this is like a quick _warning_ that something goes wrong